config/serverless-operator.yaml

```diff
@@ -13,14 +13,13 @@ config:
       - onDemand: true
         version: "4.13"
     release-1.35:
-      prowgen:
-        enabled: false
```

Comment:
@mgencur I guess, this is not intended to be removed? Probably because Go default value for boolean is false and thus "can be removed". Maybe rename it to `disabled: true`?

Reply:
No. It should stay there. No idea why this is being removed at this point.

Reply:
Probably as "false" is the default for Golangs boolean values

Reply:
That's what I meant with renaming it to "disabled" and requiring a "true" value

Reply:
This probably needs some additional work here: https://github.com/openshift-knative/hack/blob/main/pkg/discover/discover.go#L97
Looking into it.

Reply:
I think I will just rename it as you suggested.

Reply:
Sent in https://github.com/openshift-knative/hack/pull/294